src/components/ArticleList.jsx

```diff
@@ -9,12 +9,13 @@ class ArticleList extends Component {
     axios.get("/articles").then(response => {
       this.props.dispatch({
         type: "ARTICLES",
-        payload: { articleList: response.data.articles }
+        payload: { articleList: response.data }
       });
+
```

Comment:
this space may be an extra one that can be deleted

Reply:
fixed